openmetrics/datadog_checks/openmetrics/data/conf.yaml.example

```diff
@@ -88,67 +152,265 @@ instances:
     #
     # send_monotonic_counter: true
 
-    ## @param send_distribution_counts_as_monotonic - boolean - optional - default: false
+    ## @param send_distribution_counts_as_monotonic - boolean - optional - default: true
```

Comment:
Is this default change intended? The default seems to be false: https://github.com/DataDog/integrations-core/blob/3ab723d812fc79365ed4ada7e381d0f866813cc9/datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py#L201-L206